src/main/datatypes/MQ2GroupType.cpp

```diff
@@ -455,6 +750,64 @@ bool MQ2GroupType::GetMember(MQVarPtr VarPtr, const char* Member, char* Index, M
 			}
 		}
 		return false;
+		
+	case GroupMembers::Furthest:
+		Dest.DWord = 0;
+		Dest.Type = pFloatType;
+
+		for(auto& member : *pLocalPC->Group)
+		{
+			if(member && member->pSpawn)
+			{
+				auto distance = Distance3DToSpawn(pLocalPC->pSpawn, member->pSpawn);
+				if(distance > furthest_dist)
+					furthest_dist = distance;
+			}
+		}
+		Dest.Float = furthest_dist;
+		return true;
+
+	case GroupMembers::FurthestMelee:
```

Comment:
I'm a little confused why these need to be datatypes, this seems really simple to do in scripts